employee-management-client/src/app/clients/client-list/client-list.component.ts

```diff
@@ -0,0 +1,80 @@
+import { Component, Input, OnInit, ViewChild } from '@angular/core';
+import { MatDialog, MatDialogConfig } from '@angular/material/dialog';
+import { MatPaginator } from '@angular/material/paginator';
+import { MatSort } from '@angular/material/sort';
+import { MatTableDataSource } from '@angular/material/table';
+import { ClientService } from 'src/app/shared/client.service';
+import { ClientComponent } from '../client/client.component';
+
+@Component({
+  selector: 'app-client-list',
+  templateUrl: './client-list.component.html',
+  styleUrls: ['./client-list.component.scss']
+})
+export class ClientListComponent implements OnInit {
+
+  constructor(private service: ClientService,
+    private dialog: MatDialog) {
+
+  }
+  listData!: MatTableDataSource<any>;
```

Comment:
Rather than using **any** as type, create interface which will hold the same fields as table columns